plugins/whoami.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: whoami
+spec:
+  version: "v0.0.2"
```

Comment:
nit: you might wanna modify your tool/template to not to add unnecessary quotes maybe.
makes it harder to read. I'll leave it up to your to fix.

Reply:
fixed both the tool and the current PR.